huaweicloud/services/acceptance/iec/data_source_huaweicloud_iec_flavors_test.go

```diff
@@ -53,11 +53,11 @@ func testAccCheckIECFlavorsDataSourceID(n string) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
 		rs, ok := s.RootModule().Resources[n]
 		if !ok {
-			return fmtp.Errorf("Root module has no resource called %s", n)
+			return fmt.Errorf("root module has no resource called %s", n)
 		}
 
 		if rs.Primary.ID == "" {
-			return fmtp.Errorf("IEC flavors data source ID not set")
+			return fmt.Errorf("iec flavors data source ID not set")
```

Comment:
IEC is the cloud service name, should be capitalized. please check them all.
```suggestion
			return fmt.Errorf("IEC flavors data source ID not set")
```